### ISSUE TITLE
Pull in backbone v1.3.3

### ISSUE THREAD

**Issue by jeremymlane** [typhonjs/backbone-es6#2]
Any updates on the progress of this? I'd like to start using this right away, but can't until it has the latest Backbone updates.

**Comment by typhonrt** [comment]
Thanks for following up!

I'm definitely getting back to the backbone-es6 refactor and update in the coming week or so. I'm just finishing up a lot of documentation / [ESDoc plugins](https://github.com/typhonjs-node-esdoc) that took longer than expected before starting the update / refactor of backbone-es6. I plan to update the current backbone-es6 version to 1.3.3 first and get a complete test suite together before the further refactor process of splitting it into more discrete independent modules / repos. All of this should be done by mid-May or thereabout. 

Are there any particular Backbone features added between 1.2.3 and 1.3.3 that are pertinent for your usage? I wished they had detailed changelog / release notes, but the only pertinent details to review is the [commit log](https://github.com/jashkenas/backbone/commits/master) and subsequently code diffs when I update the code. Several of those releases ala 1.3.0, 1.3.1 1.3.2 were because of botched releases on NPM seemingly and perhaps a few bug / regression fixes. A lot of the changes are for documentation / tests with very few feature changes / additions. It'd be handy if you can point out any specific commit that is a feature change that you depend on presently. 

I'm definitely committed to updating backbone-es6 and related repos / efforts, so if it's just a question on whether future maintenance will occur and there are no blockers from any changes between Backbone 1.2.3 and 1.3.3 you're safe with starting to work with backbone-es6 as it's going to get done soon.  

How are you consuming backbone-es6? Are you using JSPM / SystemJS by chance or one the bundled distributions?

**Comment by wujekbogdan** [comment]
@typhonrt 
I've been looking for the Backbone changelog recently and I found this: https://allmychanges.com/p/javascript/backbone/

It's not an official changelog, though it looks reliable.

**Comment by typhonrt** [comment]
Thanks @wujekbogdan as that link does look reliable as I've been following the issues on the Backbone repo. Good find! I'll still be relying on code diffs for the update, but it's handy to see a changelog too. I'm still working on the ESDoc plugins for at least a week or thereabout, but I'll get on the backbone-es6 update right after that.

**Comment by typhonrt** [comment]
So backbone-es6 has been updated to 1.3.3 and everything seems to work at least with the canonical TODOs example. It seems like [Backbone 1.4](https://github.com/jashkenas/backbone/issues/4028) may soon be released, so I am going to postpone creating the test suite and further modularization of backbone-es6 until this release occurs. I'll leave this issue open for a week or so for any further comments.  If by chance there are any issues with the backbone-es6 1.3.3 parity update please report them as separate issues.
